src/core.js

```diff
@@ -10,23 +10,21 @@ var CryptoJS = CryptoJS || (function (Math, undefined) {
      *
      * As Math.random() is cryptographically not safe to use
      */
-    var secureRandom = function () {
+    var cryptoSecureRandomInt = function () {
         // Native crypto module on NodeJS environment
         try {
-            // Crypto from global object
-            var crypto = global.crypto;
+            // Native rypto from global object or import via require
+            var crypto = global.crypto || require('crypto');
 
-            // Create a random float number between 0 and 1
-            return Number('0.' + crypto.randomBytes(3).readUIntBE(0, 3));
+            return crypto.randomBytes(4).readInt32LE();
         } catch (err) {}
 
         // Native crypto module in Browser environment
         try {
             // Support experimental crypto module in IE 11
             var crypto = window.crypto || window.msCrypto;
 
-            // Create a random float number between 0 and 1
-            return Number('0.' + window.crypto.getRandomValues(new Uint32Array(1))[0]);
+            return (crypto.getRandomValues(new Uint32Array(1))[0]) | 1;

```

Comment:
Why `| 1`? That will make it so the returned number will always be, therefore making this not a 32 bit random number but only 31.